src/HIE/Bios/Types.hs

```diff
@@ -73,16 +73,16 @@ data CradleLoadResult r
   deriving (Functor, Show)
 
 
-data CradleError = CradleError ExitCode [String] deriving (Show)
+data CradleError = CradleError [FilePath] ExitCode [String] deriving (Show)
```

Comment:
This needs to be turned into a full record with documentation what the different fields denote